src/PrestaShopBundle/Resources/config/services/adapter/title.yml

```diff
@@ -11,12 +29,34 @@ services:
 
   prestashop.adapter.title.command_handler.delete_title_handler:
     class: 'PrestaShop\PrestaShop\Adapter\Title\CommandHandler\DeleteTitleHandler'
+    autowire: true
+    arguments:
+      $tmpImageDir: !php/const _PS_TMP_IMG_DIR_
+      $genderDir: !php/const _PS_GENDERS_DIR_
```

Comment:
Isn't it weird to inject these values even in handlers that don't need them just because they all extend the same abstract handler?

Reply:
Removed. Thanks ;)